packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -87,6 +87,7 @@ class SharingCollection extends DocumentCollection {
    * @param {Array<Recipient>=} params.readOnlyRecipients Recipients to add to the sharings with only read only access
    * @param {boolean=} params.openSharing If someone else than the owner can add a recipient to the sharing
    * @param {string=} params.appSlug Slug of the targeted app
+   * @param {boolean=} params.sharedDrive If the sharing is a shared drive
```

Comment:
A "shared drive"  must be interpreted as a "sharing by federation", in our terminology ?

Reply:
Exactly. Is is also the naming used on cozy-stack.

Reply:
Is it documented somewhere? This naming is not very intuitive from my point of view

Reply:
Is it being documented here https://github.com/cozy/cozy-stack/pull/4511

For the moment we (@nono and me) still think that shared drive is the best terminology. Federated sharing makes us think about federated services like Mastodon.

But we are still open to change.